### DISCUSSION TITLE
How do I build the drug ADME process inside PK-sim when there are several drugs?

### DISCUSSION THREAD

**Discussion by hong77777** [Open-Systems-Pharmacology/Forum#1024] [Questions & Problems] [answered]
In PK-sim to build a drug metabolic network, precursor drugs are all converted into active forms, that the precursor only need to enter the absorption and distribution plate, the active form input distribution、elimination and  excretion of plate parameters is it?

**Comment by StephanSchaller** [comment, marked as answer]
If you want to follow plasma PK of both, prodrug and active form, you implement both, prodrug and active form as a compound in PK-Sim.
You then set up a metabolic process for the pro-drug to be converted into its active form.

**Reply by hong77777** [reply]
Okay, thanks
